### PR TITLE
Add zwave_mqtt base

### PR DESCRIPTION
**Pull request by MartinHjelmare**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/34987
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1545
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/zwave_mqtt.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### Service `zwave_mqtt.add_node`
+
+This service will set the controller into inclusion mode and should be used to add a device (node) to the Z-Wave network. Call the service and then perform the device specific procedure, according to the device manual, to add your device to the network. Make sure the controller is connected to the host where the QT-OpenZwave daemon is running, when performing this operation.
+
+| Service Data Attribute | Required | Description                                                                                                                                                                                                               |
+| ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `secure`               | no       | Add the new node with secure communications. Secure network key must be set. This process will fallback to add_node (unsecure) for unsupported devices. Note that unsecure devices can't directly talk to secure devices. |
```
Comment (cgarwood) [inline source/_integrations/zwave_mqtt.markdown]: Should we mention that the network key as to be set in qt-openzwave? In the previous component it was configured in HA's config, but in this component it's configured on the qt-openzwave side

**Comment by cgarwood** [comment]
One comment regarding network key in the service description, but otherwise looks good!

File: source/_integrations/zwave_mqtt.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_integrations/zwave_mqtt.markdown]: Yeah, could be good. Should we mention this under `Configuration` and then point to that here in the service description?

Comment (MartinHjelmare) [reply source/_integrations/zwave_mqtt.markdown]: Commit done.

File: source/_integrations/zwave_mqtt.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: Z-Wave over MQTT
+description: Instructions on how to integrate Z-Wave over MQTT with Home Assistant.
+ha_category:
+  - Switch
+ha_release: 0.110
```
Comment (klaasnicolaas) [inline source/_integrations/zwave_mqtt.markdown]:
```suggestion
ha_release: "0.110"
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/zwave_mqtt.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+ha_domain: zwave_mqtt
+---
+
+This integration allows you to utilize OpenZWave's qt-openzwave daemon to control a Z-Wave network over MQTT.
+
+## Requirements
+
+- MQTT server and the [MQTT integration](/integrations/mqtt/) set up in Home Assistant
+- [qt-openzwave daemon](https://github.com/OpenZWave/qt-openzwave)
+- Supported Z-Wave dongle compatible with OpenZWave 1.6. See this [list](https://www.home-assistant.io/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules) of controllers.
```
Comment (MartinHjelmare) [inline source/_integrations/zwave_mqtt.markdown]: We should make a relative link instead.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by fabaff — APPROVED** [review]

**Pull request merged**